### ISSUE TITLE
R-BERT 구현 및 리딩

### ISSUE THREAD

**Issue by greekFire21** [boostcampaitech3/level2-klue-level2-nlp-11#10]
## 기능 추가
필요한 기능을 적어주세요.

## 기능 설명
R-BERT: Relation classfication을 위해 고안된 BERT 모델 (https://arxiv.org/abs/1905.08284)

'국립국어원 2021년 인공지능 언어능력평가'에서 1등을 차지한 Team BC의 솔루션을 참고한 바에 따르면, KLUE/Roberta-large, monologg/koelectra 등의 pretrained model을 바탕으로 구축하였다. (https://github.com/NIKL-Team-BC/NIKL-KLUE)

1) 문장 s에서 두 개의 타겟 엔티티 e1, e2의 양 옆에 스페셜 토큰을 삽입 -> 타겟 엔티티의 위치 포착
2) 문장 s가 BERT를 통과한 결과 H에서 e1과 e2의 결과 벡터를 파악 -> multi-layer neural network의 입력값
3) 두 타겟 엔티티에 대해 average operation 적용 (각 벡터 값의 합을 그 벡터의 개수로 나눠준다.)
4) H와 e1, e2가 활성화 함수(tanh)를 거친다 (-1에서 1 사이의 값)
5) fully-connected layer
6) 문장과 두 타겟 엔티티가 위의 과정을 거친 출력 벡터를 concat
7) fully-connected layer
8) softmax

## 추가 내용
위에 링크를 올린 Team BC의 코드를 참고하여 엔티티 간 관계 분류를 위한 모델을 만들어 보고자 합니다.
